src/output_counties.R

```diff
@@ -261,14 +260,14 @@ cat('\\begin{tabular}{rr@{\\hskip .3in}ccc@{\\hskip .4in}ccc@{\\hskip .4in}ccc}\
 cat('\\hline\n')
 cat('& & ',paste0('\\multicolumn{3}{c}{N=',nVec,'}',collapse='&'),'\\\\\n')
 cat('\\hline\n')
-cat('&  ',rep('&Freq & MD & EM',3),'\\\\\n')
+cat('&  ',rep('&Frequency & Min Dist & Max Likelihood',3),'\\\\\n')
 cat('\\hline\n')
 for(j in 1:length(matVec)){
-    cat('&Bias &', paste0(sapply(c(100,500,1000),function(n) paste0(sapply(c('naive','md','em'),
+    cat('&Bias &', paste0(sapply(nVec,function(n) paste0(sapply(c('naive','md','em'),
                                                                            function(e) outFunc(get(matVec[j]),n,e,'bias',tVec[j])),collapse='&')),collapse='&'),'\\\\\n')
-    cat('$',nmVec[j],'$& s.d. &', paste0(sapply(c(100,500,1000),function(n) paste0(sapply(c('naive','md','em'),
+    cat('$',nmVec[j],'$& s.d. &', paste0(sapply(nVec,function(n) paste0(sapply(c('naive','md','em'),
                                                                                           function(e) outFunc(get(matVec[j]),n,e,'sd',tVec[j])),collapse='&')),collapse='&'),'\\\\\n')
-    cat('&RMSE &', paste0(sapply(c(100,500,1000),function(n) paste0(sapply(c('naive','md','em'),
```

Comment:
@rosenbat2 maybe the problem was that `c(100,500,1000)` was hardcoded and did not include 10k?